### PR TITLE
Feature/measure backup time

### PR DESCRIPTION
**Pull request by mkryva**

### CODE REVIEW

File: twindb_backup/__init__.py [hunk 1]
```diff
@@ -168,3 +169,25 @@ def get_timeout(run_type):
         'yearly': 365 * 24 * 3600 / 2
     }
     return timeouts[run_type]
+
+
+def save_measures(start_time, end_time):
```
Comment (akuzminsky) [inline twindb_backup/__init__.py]: Pass log file as argument. It's ok to use the global variable as default value

**Comment by akuzminsky** [comment]
Please add unit tests

File: tests/unit/test_twindb_backup.py [hunk 2]
```diff
@@ -264,3 +264,30 @@ def test_get_lsn(error_log, lsn, tmpdir):
...
+        """
+{"measures": [{"duration": 444, "start": 444, "finish": 888}]}
+        """
+
+    )
+])
+def test_save_measures_if_log_is_exist(data, tmpdir):
+    log_file = tmpdir.join('log')
+    log_file.write(data)
+    save_measures(50, 100, str(log_file))
```
Comment (akuzminsky) [inline tests/unit/test_twindb_backup.py]: What exactly do you verify here?

File: twindb_backup/__init__.py [hunk 3]
```diff
@@ -168,3 +169,25 @@ def get_timeout(run_type):
...
+
+
+def save_measures(start_time, end_time, log_path=LOG_FILE):
+    """Save backup measures to log file"""
+    data = {}
+    data['start'] = start_time
+    data['finish'] = end_time
+    data['duration'] = end_time - start_time
+    if os.path.isfile(log_path):
+        with open(log_path) as data_fp:
```
Comment (akuzminsky) [inline twindb_backup/__init__.py]: I'm not sure, but if I remember correctly open() in 'a+' mode will create a file if it doesn't exist. So you can get rid of `if...else`.

Comment (mkryva) [reply twindb_backup/__init__.py]: 'a+' is only create file if file doesn't exist or append to file and open return last position in file, that will be empty for json loads.

**Pull request merged**